src/filters/interframe.cpp

```diff
@@ -163,6 +164,7 @@ static void SmartIB_MMX(uint8_t *srcPtr, uint32_t srcPitch, int width, int start
 
 void SmartIB(uint8_t *srcPtr, uint32_t srcPitch, int width, int starty, int height)
 {
+  width = width;
```

Comment:
Again, if there is no way to fix this warning it would probably better to just turn it off, I can do that just tell me what the warning is.

Reply:
Just as stated before, we can `__attribute__((unused))` this.
```
/home/denisfa/Git/visualboyadvance-m/src/filters/interframe.cpp: In function ‘void SmartIB(uint8_t*, uint32_t, int, int, int)’:
/home/denisfa/Git/visualboyadvance-m/src/filters/interframe.cpp:164:54: warning: unused parameter ‘width’ [-Wunused-parameter]
 void SmartIB(uint8_t *srcPtr, uint32_t srcPitch, int width, int starty, int height)
                                                  ~~~~^~~~~
```